### DISCUSSION TITLE
Embedded database dumpt and restore in dedicated one

### DISCUSSION THREAD

**Discussion by zeldigas** [tolgee/tolgee-platform#2016] [Q&A] [answered]
I deployed tolgee as a service in k8s with defaults that assumed embedded DB usage. It works fine, I already created some data there, but obviously with deployment recreation data will be lost.

Is it possible to dump all information somehow (e.g. as sql db dump or via api) and then upload it to another tolgee instance that will be created with persistant data storage?

I'm insterested in preserving projects with all the data, users, api tokens.

**Comment by JanCizmar** [comment, marked as answer]
Hello! You're definitely able to dump the data. The embedded database is Postgres. So you should be able to access it by forwarding the 5432 port of the pod.

**Reply by zeldigas** [reply]
Thank you! Will try this
